grafana/grafana.bzl

```diff
@@ -68,26 +69,41 @@ py_dashboard = rule(
     },
 )
 
-def _py_dashboard_builder(src, deps=None):
+def _py_dashboard_builder(src, python_version = "PY3", deps=None):
```

Comment:
i think it'd be clearest and easiest for upgrading if this didn't have a default, so it has to be provided